templates/base.html

```diff
@@ -35,12 +37,15 @@
 			</div>
 			{% block search %}{% endblock %}
 			<div id="links">
-				<a id="reddit_link" href="https://www.reddit.com{{ url }}" rel="nofollow">
+				<a id="reddit_link" {% if prefs.visit_reddit_confirmation != "on" %}href="#popup"{% else %}href="https://www.reddit.com" rel="nofollow"{{ url }}"{% endif %}>
```

Comment:
```suggestion
				<a id="reddit_link" {% if prefs.disable_visit_reddit_confirmation != "off" %}href="#popup"{% else %}href="https://www.reddit.com" rel="nofollow"{{ url }}"{% endif %}>
```

Reply:
i see i've missed a few to rename, will rename the last in a separate commit. but switching from `"on"` to `"off"` will create a logical error where the setting for disabling the popup doesn't work properly